src/QsCompiler/Tests.LanguageServer/Tests.cs

```diff
@@ -66,12 +66,22 @@ public async Task InitializationAsync()
             var initParams = TestUtils.GetInitializeParams();
             var initReply = await this.rpc.InvokeWithParameterObjectAsync<InitializeResult>(Methods.Initialize.Name, initParams);
             Assert.IsNotNull(initReply);
+            Assert.IsFalse(this.server.IsNotebook);
 
             var init = await this.rpc.InvokeWithParameterObjectAsync<JToken>(Methods.Initialize.Name, initParams);
             var initializeError = Utils.TryJTokenAs<InitializeError>(init);
             Assert.IsTrue(initializeError != null ? initializeError.Retry : false);
         }
 
+        [TestMethod]
+        public async Task NotebookInitializationAsync()
+        {
+            var initParams = TestUtils.GetInitializeParams(addNotebookConfig: true);
+            var initReply = await this.rpc.InvokeWithParameterObjectAsync<InitializeResult>(Methods.Initialize.Name, initParams);
+            Assert.IsNotNull(initReply);
+            Assert.IsTrue(this.server.IsNotebook);
+        }
+
```

Comment:
Are there other unit tests possible? I don't want to mess up coverage. I struggled to find tests for all the Managers, but maybe I wasn't looking hard enough

Reply:
Added some more in a couple of new commits